src/pages/PostPage.scss

```
@@ -0,0 +1,26 @@
+@import '../assets/styles/common.scss';
+@import '../assets/styles/palette.scss';
+@import '../assets/styles/fonts.scss';
+
+.page-container {
+  &.post {
+    padding: 57px 600px;
+  }
+}
+TextInputField {
+  width: 720px;
+}
+.font-24-bold {
+  @include font-style(24, bold);
+  color: $gray-900;
+}
```

```suggestion
.font-24-bold {
  color: $gray-900;
}
```

fonts.scss 보시면 .font-24-bold에는 이미 size랑 weight는 설정되어있어용 ~~ ㅎㅎ

---

아차차 이 부분 테스트하고 지운다는 것을 깜빡했네요🤦🏻‍♀️...